src/mdx-pages/getting-started.mdx

```diff
@@ -11,7 +11,7 @@ Our friends at [Fastly](http://fastly.com) are nice enough to provide hosting fo
   <link href="https://vjs.zencdn.net/${VJS_VERSION}/video-js.css" rel="stylesheet" />
 
   <!-- If you'd like to support IE8 (for Video.js versions prior to v7) -->
-  <script src="https://vjs.zencdn.net/ie8/1.1.2/videojs-ie8.min.js"></script>
+  <!-- <script src="https://vjs.zencdn.net/ie8/1.1.2/videojs-ie8.min.js"></script> -->
```

Comment:
Making it a comment makes sense. Though, definitely confused me for a second when I checked out https://deploy-preview-125--videojs-dot-com.netlify.app/getting-started.